sdk/eventhub/azure-eventhub/azure/eventhub/_constants.py

```diff
@@ -44,6 +44,8 @@
 MGMT_STATUS_CODE = b"status-code"
 MGMT_STATUS_DESC = b"status-description"
 USER_AGENT_PREFIX = "azsdk-python-eventhubs"
+UAMQP_TRANSPORT = "uamqp"
+PYAMQP_TRANSPORT = "pyamqp"
```

Comment:
since we use `uamqp_transport` as a bool in lots of other places, to avoid confusion, maybe we can call these `UAMQP_USER_AGENT_PREFIX`/`PYAMQP_USER_AGENT_PREFIX` or something similar?